VCFv4.3.tex

```diff
@@ -253,6 +253,17 @@ \subsubsection{Pedigree field format}
 ##pedigreeDB=URL
 \end{verbatim}
 
+
+\subsubsection{Reference block checkpoint}
+Given the ability to interpret missing genotypes as either truly missing or as part of a reference block (see RBS in Genotype tags below), it can be useful to limit the genomic distance required to scan in order to find the "top" of the reference block. 
+To enable this, one may specify a Reference Block Checkpoint scheme:
+\begin{verbatim}
+##REFERENCE_BLOCK=<CHECKPOINT=1000>
```

Comment:
This is invalid as a structured meta-information line, because it does not have an `ID=…` subfield.

Are there plans to expand this in future with additional subfields within the `<…>`? If not, I'd suggest reworking this as a plain unstructured meta-information line, e.g.,

```
##REFERENCE_BLOCK_CHECKPOINT=1000
```